lib/network/modules/Groups.js

```diff
@@ -75,14 +75,17 @@ class Groups {
   }
   
   /**
-   * get group options of a groupname. If groupname is not found, a new group
-   * is added.
-   * @param {*} groupname        Can be a number, string, Date, etc.
-   * @return {Object} group      The created group, containing all group options
+   * Get group options of a groupname.
+   * If groupname is not found, a new group may be created.
+   *
+   * @param {*}       groupname     Can be a number, string, Date, etc.
+   * @param {boolean} [create=true] If true, create a new group
+   * @return {Object} The found or created group
    */
-  get(groupname) {
+  get(groupname, create = true) {
```

Comment:
I personally like more declarative variables. I think it should be: `create` => `shouldCreate`

Reply:
:+1: